bb/logmsg.c

```diff
@@ -48,6 +48,8 @@ static int level_to_syslog(loglvl lvl) {
             return LOG_DEBUG;
         case LOGMSG_INFO:
             return LOG_INFO;
+        case LOGMSG_USER:
+            return LOG_USER;
```

Comment:
Ok, from man syslog: The priority argument is formed by ORing the facility and the level values. 
So SYS_USER is a facility...I understand now. Still, returning CRITICAL is probably not good idea, so should return something else, maybe LOG_INFO.